app/config.go

```diff
@@ -258,7 +269,8 @@ func InitGenesis() {
 func InitTendermint() *node.Node {
 	logger := log.NewTMLoggerWithColorFn(log.NewSyncWriter(os.Stdout), func(keyvals ...interface{}) term.FgBgColor {
 		if keyvals[0] != kitlevel.Key() {
-			panic(fmt.Sprintf("expected level key to be first, got %v", keyvals[0]))
+			fmt.Println(fmt.Sprintf("expected level key to be first, got %v", keyvals[0]))
+			os.Exit(1)
```

Comment:
```suggestion
			log.Fatalf("expected level key to be first, got %v", keyvals[0])

```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)

the `log` package needs to also be added on this file for this suggestion to work